src/infra/src/table/short_urls.rs

```diff
@@ -131,10 +128,7 @@ pub async fn add(short_id: &str, original_url: &str) -> Result<(), errors::Error
     let _lock = get_lock().await;
 
     let client = ORM_CLIENT.get_or_init(connect_to_orm).await;
-    Entity::insert(record)
-        .exec(client)
-        .await
-        .map_err(|e| Error::DbError(DbError::SeaORMError(e.to_string())))?;
+    Entity::insert(record).exec(client).await?;
```

Comment:
This logic moved to [this trait implementation](https://github.com/openobserve/openobserve/pull/5100/files#r1847373471) so it should now happen automatically when `?` converts the error.